superset-frontend/src/chart/Chart.jsx

```diff
@@ -175,7 +186,6 @@ class Chart extends React.PureComponent {
 
   render() {
     const {
-      width,
```

Comment:
It looks like the width was used in a few places and it is also defined in the PropTypes. Would you like to check where it is used and delete it if you deem it unnecessary?

Reply:
Yes. I'll do that.